packages/ra-ui-materialui/src/field/ReferenceField.stories.tsx

```diff
@@ -837,3 +837,60 @@ const AccessControlUI = ({
         </div>
     );
 };
+
+export const Nested = () => (
```

Comment:
This story was added to test the nested case, but the bug isn't there. Nonetheless, the story is useful to show that it's working!